src/js/data-store.js

```diff
@@ -45,30 +45,34 @@ async function fetchItems (itemIDs) {
 		body: JSON.stringify(itemIDs)
 	};
 
-	try {
-		const response = await fetch(`/syndication/resolve${location.search}`, options);
+	if(options.body.length) {
```

Comment:
Should this be `itemIDs.length`?  An empty array will generate the string `[]` which has a length of 2, or `JSON.stringify(null)` will generate the string `"null"`.

Reply:
Yes, yes it should. I'm sorry I didn't notice this earlier, it would've saved me some head-scratching!

Fixed here: https://github.com/Financial-Times/n-syndication/pull/65